marketplaceItems/audioFocus/marketplaceResources/Description.md

```diff
@@ -0,0 +1,19 @@
+# AUDIO FOCUS
+
+Having trouble hearing in a noisey room?
+
+Domain attenuation getting you down?
+
+Feeling left out of the conversation?
+
+Introducing:
+
+## AUDIO FOCUS
+
+once again that's 
+
+# AUDIO FOCUS!!!!!
+
+Just open the app, then click(desktop) or laser(HMD) on an avatar to hear just them.
```

Comment:
"just" used twice in one sentence - perhaps replace the final "just" with "only" ?